Initialtask/src/Validator/ProductValidator.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Validator;
+
+use App\Entity\Product;
+
+class ProductValidator
+{
+    public static function validate(array $productData): bool
+    {
+        $cost = (float) preg_replace('/[^0-9.]/', '', $productData['Cost in GBP']);
+        $stok = (int) preg_replace('/[^0-9]/', '', $productData['Stock']);
+        if (self::isValidMinCostAndStock($cost, $stok) && self::isValidMaxCost($cost)) {
```

Comment:
Please, configure cs-fixer correctly
Should be empty line before and after if clause, for readability